src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/ApprenticeshipNeverStartedViewModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.Authorization.ModelBinding;
+using System.ComponentModel;
+using System;
```

Comment:
order system usings first. (Update VS settings to always do this)

Reply:
Done both in settings and files.